ViAn/GUI/recentprojectdialog.cpp

```diff
@@ -40,6 +41,12 @@ RecentProjectDialog::RecentProjectDialog(QWidget* parent) : QDialog(parent) {
     connect(open_btn, &QPushButton::clicked, this, &RecentProjectDialog::on_open_btn_clicked);
 }
 
+RecentProjectDialog::~RecentProjectDialog() {
+    delete h_layout;
```

Comment:
Each gui component will be deleted automatically by Qt if parented correctly. This isn't the case here but in future implementations we should set the parent correctly so we won't have to worry about cleanup